microsoft-edge/webdriver-chromium.md

```diff
@@ -56,6 +56,36 @@ The last component you must download is a language-specific client driver to tra
 
 If you already use [Selenium 3][SeleniumHQ], you may have existing browser tests and want to add coverage for Microsoft Edge \(Chromium\) without changing your version of Selenium.  To use [Selenium 3][SeleniumHQ] to write automated tests for both Microsoft Edge \(EdgeHTML\) and Microsoft Edge \(Chromium\), install the [Selenium Tools for Microsoft Edge][GithubMicrosoftEdgeSeleniumTools] package to use the updated driver.  The `EdgeDriver` and `EdgeDriverService` classes included in the tools are fully compatible with the built-in equivalents in Selenium 4.  
 
+Install the [Selenium Tools for Microsoft Edge][GithubMicrosoftEdgeSeleniumTools] and [Selenium 3][SeleniumHQ].
```

Comment:
@zoherghadyali - do customers need to do this step, or are all the steps listed in the tabbed boxes that follow? If the steps are in the tabbed boxes below, then maybe we change this from "install..." to "For more information, see..."?

Reply:
I updated to: "Use the following steps to add the Selenium Tools for Microsoft Edge and Selenium 3 to your project."

If you have a better idea for less awkward phrasing, please suggest it, I'm running into writer's block with this one sentence.